fedify/runtime/docloader.ts

```diff
@@ -311,7 +326,13 @@ export function getDocumentLoader(
   { allowPrivateAddress, skipPreloadedContexts, userAgent }:
     GetDocumentLoaderOptions = {},
 ): DocumentLoader {
-  async function load(url: string): Promise<RemoteDocument> {
+  async function load(
+    url: string,
+    _options?: DocumentLoaderOptions,
+  ): Promise<RemoteDocument> {
+    if (_options?.signal?.aborted) {
+      throw new DOMException("Aborted", "AbortError");
+    }
```

Comment:
You could use [`AbortSignal.throwIfAborted()`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/throwIfAborted) method instead.